### PR TITLE
When you click on the logo it goes to the dashboard

### PR DESCRIPTION
**Pull request by chaimbinder**
# Description
<Please describe the suggested changes>
When you click on the logo it goes to the dashboard #229

### CODE REVIEW

**Review by shootermv — CHANGES_REQUESTED** [review]

File: src/layout/sidebar/SideBar.tsx
[the remark this answers is not in the document]
Comment (giladfuchs) [reply src/layout/sidebar/SideBar.tsx:1]: i suggest adding        
i18 instead Hebrew hardcoded

`{t(TEXT_KEYS.website_name)}`

**Comment by NoamGaash** [comment]
Congrats for your first contribution!
Thanks for joining us.

**Review by NoamGaash — APPROVED** [review]
As I see it, this code can be merged.
Please let us know if you want to make the suggested changes, or let it be merged as-is.
Thanks, and welcome :partying_face: :balloon: 
Noam

File: src/layout/sidebar/SideBar.tsx
[the remark this answers is not in the document]
Comment (NoamGaash) [reply src/layout/sidebar/SideBar.tsx:1]: the `TEXT_KEYS` constant should be removed
```js
{t(`website_name`)}
```

**Review by shootermv — APPROVED** [review]

**Pull request merged**